staging/src/k8s.io/component-base/logs/registry.go

```diff
@@ -103,4 +107,5 @@ func init() {
 	// Text format is default klog format
 	logRegistry.Register(defaultLogFormat, nil)
 	logRegistry.Register(jsonLogFormat, json.JSONLogger)
+	Formats = fmt.Sprintf(`"%s"`, strings.Join(logRegistry.List(), `", "`))
```

Comment:
logRegistry is meant for allowing runtime registering of formats. Creating list of formats in init is to early. We should not generate list of formats until formats.frozen is called.

Reply:
Rollback to hardcode for log format.